### PR TITLE
getlantern/lantern#2174 Made connection pooling on fronted optional

### PR DESCRIPTION
**Pull request by oxtoacart**

### CODE REVIEW

File: src/github.com/getlantern/fronted/dialer.go [hunk 1]
```diff
@@ -104,13 +112,21 @@ func NewDialer(cfg *Config) *Dialer {
 	if d.Masquerades != nil {
 		d.masquerades = d.verifiedMasquerades()
 	}
-	d.connPool = &connpool.Pool{
-		MinSize:      30,
-		ClaimTimeout: idleTimeout,
-		Dial:         d.dialServer,
+	if cfg.PoolSize > 0 {
+		d.connPool = &connpool.Pool{
+			MinSize:      30,
```
Comment (myleshorton) [inline src/github.com/getlantern/fronted/dialer.go]: Looks like this should be cfg.PoolSize

**Comment by myleshorton** [comment]
Looks good aside from that one issue.

File: src/github.com/getlantern/fronted/dialer.go (hunk 1 above)
Comment (oxtoacart) [inline src/github.com/getlantern/fronted/dialer.go]: Good catch!  Fixed.

**Comment by oxtoacart** [comment]
Already included in https://github.com/getlantern/flashlight-build/pull/34

**Pull request closed without merging**